libs/search-widget/src/widgets/search-input/SearchInput.svelte

```diff
@@ -38,13 +38,15 @@
   });
 
   $: {
-    if (value.trim() !== previous.trim()) {
+    const decodedValue = decodeURIComponent(value);
```

Comment:
Maybe it's not safe to use `decodeURIComponent` without a try catch.
Yesterday Eric fixed a similar bug related to it in #207

Reply:
You're right. Also Eric explained me the expected behavior of the suggestion so I'll rework the PR. This decode was actually not needed and will be removed in my next commit